src/credentials/sso/diskCache.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import * as crypto from 'crypto'
+import { SsoCache } from './ssoCache'
+import { ClientRegistration } from './clientRegistration'
+import { join } from 'path'
```

Comment:
Nit: we generally use `import * as path from 'path'` since `join` isn't necessarily the most descriptive on its own.

Reply:
We should probably do `import { join as joinPath } from 'path'` * imports are icky

Reply:
Changed to import { join as joinPath } from 'path'